tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/dashboard/documentdashboard.controller.js

```diff
@@ -513,7 +513,12 @@
                     var oEmailRelativePath = '';
                     var sFileURLs = ""
                     angular.forEach(vm.selectedDocuments, function (selectedDocument) {
-                        oEmailRelativePath = trimEndChar(unescape(selectedDocument.documentUrl));
+
+                        var docUrl = selectedDocument.documentUrl;

```

Comment:
Mail cart was not working for office documents and this fix will solve that issue